res/TensorFlowPythonExamples/examples/conv2d_2/__init__.py

```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+import numpy as np
+
+in_ = tf.compat.v1.placeholder(tf.float32, shape=(1, 32, 32, 3), name="Hole")
+
+filters = np.array(
+    np.random.uniform(low=-1., high=1, size=[5, 5, 3, 32]), dtype=np.float32)
```

Comment:
(Optional) You may use ``np.random.uniform(...).astype(np.float32)`` instead ;-)

Reply:
@binarman  , apply suggestion? or merge as-is ?

Reply:
@parjong
actually you suggestion sound good

Reply:
OK, I'll wait for new changes :)